.vscode/settings.json

```diff
@@ -14,5 +14,6 @@
             "model/**/*.yaml"
         ]
     },
-    "json.schemaDownload.enable": true
+    "json.schemaDownload.enable": true,
+    "markdown.extension.toc.levels": "2..6"
```

Comment:
This fixes an autoformatting issue on save in vscode with the markdown extension.
By default the autoformat includes the h1 heading leading to changed md files (eg to include `# Semantic Conventions for CICD Metrics` in the TOC).
This change aligns the autoformatting with the `markdown-toc --no-first-h1` command.
With this change autoformatting no longer causes changed md files.